lib/internal/util/inspect.js

```diff
@@ -1698,7 +1699,7 @@ function formatProperty(ctx, value, recurseTimes, key, type, desc) {
           (ctx.getters === 'get' && desc.set === undefined) ||
           (ctx.getters === 'set' && desc.set !== undefined))) {
       try {
-        const tmp = value[key];
+        const tmp = desc.get.call(original);
```

Comment:
```suggestion
        const tmp = ReflectApply(desc.get, original, []);
```

Reply:
@aduh95 to fix this, I imported `ReflectApply` from `primordials`:
```shell
Running JS linter...

/home/runner/work/node/node/lib/internal/util/inspect.js
Error:   1702:21  error  'ReflectApply' is not defined  no-undef

✖ 1 problem (1 error, 0 warnings)

Makefile:1217: recipe for target 'lint-js' failed
make: *** [lint-js] Error 1
```
Hope this works! :crossed_fingers:

Reply:
Yes sorry, I should have added that to my previous comment! Thanks for taking the suggestion :)